docs/complex_types.md

```diff
@@ -17,34 +17,60 @@ The following list represent the avro complext types mapped to python types:
 Example:
 
 ```python
-import typing
+import enum
+from typing import Optional
 
-from dataclasses_avroschema import AvroModel, types
+from dataclasses_avroschema import AvroModel
 
 
+class FavoriteColor(enum.Enum):
+    BLUE = "Blue"
+    YELLOW = "Yellow"
+    GREEN = "Green"
+    
+    class Meta:
+        doc = "A favorite color"
+        namespace = "some.name.space"
+        aliases = ["Color", "My favorite color"]
+        default = "Green"
+
 class User(AvroModel):
     "An User"
-    favorite_colors: types.Enum = types.Enum(["BLUE", "YELLOW", "GREEN"], default="BLUE")
+    favorite_color: Optional[FavoriteColor] = None
 
 User.avro_schema()
 
-
 '{
   "type": "record",
   "name": "User",
-  "fields": [
+  "fields":
+  [
     {
-      "name": "favorite_colors",
-      "type": {
-        "type": "enum",
-        "symbols": ["BLUE", "YELLOW", "GREEN"],
-        "name": "favorite_color"
-      },
-      "default": "BLUE"
+      "name": "favorite_color",
+      "type":
+      [
+        "null",
+        {
+          "type": "enum",
+          "name": "FavoriteColor",
+          "symbols": ["Blue", "Yellow", "Green"],
+          "doc": "A favorite color",
+          "namespace": "some.name.space",
+          "aliases": ["Color", "My favorite color"],
+          "default": "Green"
+        }
+      ],
+      "default": null
     }
   ],
   "doc": "An User"
 }'
+
+```
+
+Note that in addition to a *field-level* default value, you can optionally supply a *type-level* default symbol for the 
+enum by supplying the `default` attribute under the `Meta` class.
```

Comment:
I wonder if it would be worth mentioning the Avro specification on this. What do you think?

Reply:
I think we should mention it.

Reply:
done!